content/basics/discord.md

```diff
@@ -5,38 +5,52 @@ weight = 3
+The flottbot framework also supports Discord as a remote chat network.
 
 ## Setup Your Discord Bot
 
 ### Create your bot as an app
 
-1. Navigate to the [Applications page](https://discordapp.com/developers/applications/me) and click on the [New App](https://discordapp.com/developers/applications/me/create) button.
-2. Enter your bot's name under the `APP NAME *` section.
-3. Click create the `Create App` button on the bottom right.
+1. Navigate to the [Applications page](https://discordapp.com/developers/applications/me) 
+1. Click on the `New Application` button
+1. Enter your bot's name on the next screen
+1. Click create the `Create` button on the bottom right
 
 ### Add your bot to your server
 
-1. Navigate to the [Applications page](https://discordapp.com/developers/applications/me) and select one of your bot applications.
-2. In the App Details section, copy the `Client ID` value.
-3. Replace the `CLIENTID` in the following link with your bot's client ID from step 2: `https://discordapp.com/oauth2/authorize?client_id=CLIENTID&scope=bot`
-4. Go to the link you generated, and select a server from the drop-down menu. **You must have the Manage Server permission to add a bot to a server!** If no servers appear, you may need to log in.
+1. Navigate to the [Applications page](https://discordapp.com/developers/applications/me)
+1. Select one of your bot applications
+1. Click on `OAuth2` in the side navigation under "Settings"
+1. Check the checkbox for `Bot` in the "Scopes" section
+1. Copy the link it generated and navigate to it in your browser
+1. Select a server from the drop-down menu. **You must have the Manage Server permission to add a bot to a server!** If no servers appear, you may need to log in.
 
 ## Configure Your Bot
 
-Edit the following lines in `/config/bot.yml` to look like:
+First you will need your **bot token** and **server id**. For the bot token, follow these steps:
+
+1. Navigate to the [Applications page](https://discordapp.com/developers/applications/me)
+1. Select one of your bot applications
+1. Click on `Bot` in the side navigation under "Settings"
+1. Use `Click to Reveal Token` in the "Build-A-Bot" section to see your token
+
+To get your server id, [check Discord's official knowledgebase article](https://support.discordapp.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-)
+
+Now that you have both your bot token and server id, edit the following lines in `/config/bot.yml` to look like:
 
 ```yaml
 chat_application: discord
 discord_token: ${DISCORD_TOKEN}
+discord_server_id: ${DISCORD_SERVER_ID}
 ```
 
 Write some rules for your bot.
 
-Now, you'll need to export your Discord bot token as an environment variable:
+Now, you'll need to export your Discord bot token and server id values as environment variables:
 
 ```sh
 export DISCORD_TOKEN=xxxxxxxxxxxxxxxxxx.xxxxxx.xxxxxxxxx.xxxxxxxxxx
+export DISCORD_SERVER_ID=###############
```

Comment:
Does this variable also need to be added into the `docker run` line via the `--env` switch?

Reply:
yes, absolutely.. thanks for catching that

Reply:
Updated